### PR TITLE
Update to Axe.Windows 0.3.0, remove Sarif dependencies

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
Update to Axe.Windows 0.3.0 (includes explicitly adding Newtonsoft where needed)

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights/AccessibilityInsights.csproj [hunk 1]
```diff
@@ -87,12 +87,9 @@
       <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="Sarif, Version=2.0.0.0, Culture=neutral, PublicKeyToken=21a5e83f6f5bb844, processorArchitecture=MSIL">
-      <HintPath>..\packages\Axe.Windows.0.2.0-prerelease\lib\net471\Sarif.dll</HintPath>
+      <HintPath>..\packages\Sarif.Sdk.2.0.0-csd.2.beta.2018-10-10\lib\net461\Sarif.dll</HintPath>
```
Comment (karanbirsingh) [inline src/AccessibilityInsights/AccessibilityInsights.csproj]: @DaveTryon is this one still remaining?

File: src/UITests/UITests.csproj [hunk 2]
```diff
@@ -80,15 +80,16 @@
       <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
     <Reference Include="Sarif, Version=2.0.0.0, Culture=neutral, PublicKeyToken=21a5e83f6f5bb844, processorArchitecture=MSIL">
-      <HintPath>..\packages\Axe.Windows.0.2.0-prerelease\lib\net471\Sarif.dll</HintPath>
+      <HintPath>..\packages\Sarif.Sdk.2.0.0-csd.2.beta.2018-10-10\lib\net461\Sarif.dll</HintPath>
```
Comment (karanbirsingh) [inline src/UITests/UITests.csproj]: This as well (Sarif)

File: src/AccessibilityInsights/AccessibilityInsights.csproj (hunk 1 above)
Comment (DaveTryon) [reply src/AccessibilityInsights/AccessibilityInsights.csproj]: Removed in iteration 5

---
In reply to: [303560673](https://github.com/microsoft/accessibility-insights-windows/pull/430#discussion_r303560673) [](ancestors = 303560673)

File: src/UITests/UITests.csproj (hunk 2 above)
Comment (DaveTryon) [reply src/UITests/UITests.csproj]: Removed in iteration 5

---
In reply to: [303560850](https://github.com/microsoft/accessibility-insights-windows/pull/430#discussion_r303560850) [](ancestors = 303560850)

**Pull request merged**